wlutil/build.py

```diff
@@ -389,7 +413,6 @@ def makeImage(config):
         # Apply and run the init script
         init_overlay = config['builder'].generateBootScriptOverlay(str(config['guest-init'].path), config['guest-init'].args)
         applyOverlay(config['img'], init_overlay)
-        print("Launching: " + str(config['bin']))
```

Comment:
Good catch. This used to be needed because I couldn't figure out how to supress the output of Qemu when running the guest-init and it was confusing to users. Since I fixed it, this isn't needed. Thanks.

Reply:
Oh I think this was a mistake on my end. I had other print statements that I was using that I wanted to delete so I did a blanket delete. At least this is still good!